src/components/v5/common/ActionSidebar/partials/CompletedAction/CompletedActionSidebar.tsx

```diff
@@ -0,0 +1,314 @@
+                  to={COLONY_ACTIVITY_ROUTE}
+                  className="flex-1"
+                  onClick={toggleActionSidebarOff}
+                >
+                  {formatText({
+                    id: 'actionSidebar.fourOfour.activityPageLink',
+                  })}
+                </ButtonLink>
+              ) : (
+                <Button
+                  mode="primarySolid"
+                  className="flex-1"
+                  onClick={startPollingForAction}
+                >
+                  {formatText({
+                    id: 'button.retry',
+                  })}
+                </Button>
+              )
+            }
+          />
+        </div>
+      </ActionSidebarLayout>
+    );
+  }
+
+  const actionType = getExtendedActionType(action, colony.metadata);
+
+  const getActionContent = () => {
+    switch (actionType) {
+      case ColonyActionType.Payment:
+      case ColonyActionType.PaymentMotion:
+        return <SimplePayment action={action} />;
+      case ColonyActionType.MintTokens:
+      case ColonyActionType.MintTokensMotion:
+        return <MintTokens action={action} />;
+      case ColonyActionType.MoveFunds:
+      case ColonyActionType.MoveFundsMotion:
+        return <TransferFunds action={action} />;
+      case ColonyActionType.CreateDomain:
+      case ColonyActionType.CreateDomainMotion:
+      case ColonyActionType.EditDomain:
+      case ColonyActionType.EditDomainMotion:
+        return <ManageTeam action={action} />;
+      case ColonyActionType.UnlockToken:
+      case ColonyActionType.UnlockTokenMotion:
+        return <UnlockToken action={action} />;
+      case ColonyActionType.VersionUpgrade:
+      case ColonyActionType.VersionUpgradeMotion:
+        return <UpgradeColonyVersion action={action} />;
+      case ColonyActionType.CreateDecisionMotion:
+        return <CreateDecision action={action} />;
+      case ColonyActionType.SetUserRoles:
+      case ColonyActionType.SetUserRolesMotion:
+        return <SetUserRoles action={action} />;
+      case ColonyActionType.AddVerifiedMembers:
+      case ColonyActionType.AddVerifiedMembersMotion:
+        return <AddVerifiedMembers action={action} />;
+      case ColonyActionType.RemoveVerifiedMembers:
+      case ColonyActionType.RemoveVerifiedMembersMotion:
+        return <RemoveVerifiedMembers action={action} />;
+      case ColonyActionType.EmitDomainReputationReward:
+      case ColonyActionType.EmitDomainReputationRewardMotion:
+      case ColonyActionType.EmitDomainReputationPenalty:
+      case ColonyActionType.EmitDomainReputationPenaltyMotion:
+        return <ManageReputation action={action} />;
+      case ColonyActionType.ColonyEdit:
+      case ColonyActionType.ColonyEditMotion:
+        return <EditColonyDetails action={action} />;
+      case ExtendedColonyActionType.UpdateColonyObjective:
+        return <UpgradeColonyObjective action={action} />;
+      // @TODO: Connect this to the reputation actions
```

Comment:
I have no idea why these are even commented out because they actually work (I've just commented that out on multi-sig and it just works :tada: )